wagtail/admin/templates/wagtailadmin/home.html

```diff
@@ -34,25 +34,5 @@ <h1>{% block branding_welcome %}{% blocktrans %}Welcome to the {{ site_name }} W
 
 {% block extra_js %}
 {{ block.super }}
-<script>
-$(function() {
-    $('.object.collapsible').each(function() {
-        var $target = $(this);
-        var $content = $target.find('.object-layout');
-        if ($target.hasClass('collapsed') && $target.find('.error-message').length == 0) {
-            $content.hide();
-        }
-
-        $target.find('> .title-wrapper').on('click', function() {
-            if (!$target.hasClass('collapsed')) {
-                $target.addClass('collapsed');
-                $content.hide('slow');
-            } else {
-                $target.removeClass('collapsed');
-                $content.show('show');
-            }
-        });
-    });
-});
-</script>
```

Comment:
I don't think we want to take this out of this template and load the entire `page-editor.js`. There's a lot of JS in that file that is totally useless on the Wagtail Admin homepage.

Reply:
I extracted this from another PR but it makes totally sense, sorry for this change. 
Should we continue copy-paste this snippet with a comment, with a warning in both places in order to keep these in sync ?
Or do you think it would be better to extract this in its own js file ?

Reply:
I'd be strongly in favour of moving this into its own `collapsible.js` file, and importing that both here and in wagtailadmin/pages/_editor_js.html. (I'm pretty sure that every view that currently depends on page-editor.js just includes the whole of _editor_js.html - a bad habit in the codebase that I'm trying to eliminate... - so that should preserve backwards compatibility.)

Reply:
Thanks for your feedback, I will update following these shortly

Reply:
I have updated accordingly, let me know if this need more work